pkgs/top-level/all-packages.nix

```diff
@@ -13963,11 +13963,6 @@ with pkgs;
     python = python3;
     mkDerivation = stdenv.mkDerivation;
   };
-  octave-jit = callPackage ../development/interpreters/octave {
```

Comment:
Please add an alias in `pkgs/top-level/aliases.nix` throwing an error.

Reply:
> Please add an alias in `pkgs/top-level/aliases.nix` throwing an error.

I was thinking about it, but in this case, `octave-jit` was marked as broken ever since it was introduced so I don't think it's ought an alias since there is no chance anyone ever used this derivation.

Reply:
I think the alias is still nice. It will at least document that we tried to have the JIT at some point.

Reply:
If the code was always marked broken, won't it be more misleading than useful?

Reply:
> It will at least document that we tried to have the JIT at some point.

This will be documented in Nixpkgs' Git history, and it's documented in upstream's release notes.

Reply:
Sorry, it is just not as easy to check the git history than see a comment in the code/alias explaining what is happening.

But ok, if you're fine with it I am not that against it.

Reply:
Thanks @thiagokokada for saying your opinion, it's appreciated although the disagreement. Cheers :beers: .